wasm/src/serialize.rs

```diff
@@ -0,0 +1,24 @@
+use std::rc::Rc;
+use wasm_bindgen::prelude::wasm_bindgen;
+
+use crate::flags::ALLOW_BACKREFS;
+use crate::lazy_node::LazyNode;
+use clvmr::serde::{node_from_bytes, node_from_bytes_backrefs, serialized_length_from_bytes};
+use clvmr::Allocator;
+
+#[wasm_bindgen]
+pub fn serialized_length(program: &[u8]) -> Result<u64, String> {
+    serialized_length_from_bytes(program).map_err(|x| x.to_string())
+}
+
+#[wasm_bindgen]
+pub fn sexp_from_bytes(b: &[u8], flag: u32) -> Result<LazyNode, String> {
```

Comment:
is this new? maybe `node_from_bytes` would be a more uniform name.

Reply:
Right, but `node_from_bytes` already exists.
I moved this code to `LazyNode::from_bytes()`

Reply:
The `SExp` terminology originated from me, based on words use in lisp. I admit that I did a bad job describing differentiating between `CLVMObject`, `SExp`, `Program` and `Node`, and this has made things unduly confusing.

I think using `Node` in `clvm_rs` makes sense, and maybe we should avoid creating new API that uses the term `SExp`.

Reply:
> what do you mean? is there a wasm binding for `node_from_bytes` already?

There is a `node_from_bytes` in the original source.

![image](https://github.com/Chia-Network/clvm_rs/assets/84098616/d7508be1-618e-43da-81b5-72d60dd59e8f)

I didn't like to confuse people by attaching an existing function name to a new function even if it is used in the different context (wasm).

Reply:
![image](https://github.com/Chia-Network/clvm_rs/assets/84098616/47fa3bf6-25cd-4972-9cba-501af3826758)

I think moving `node_from_bytes()` to `LazyNode::from_bytes()` looks concise and understandable.

Reply:
it seems resonable to use the same names/terminology in wasm as we do in the python binding and rust

Reply:
Ok, then using the same name is more consistent...
By the way, which way do you feel preferable?
1. `sexp_from_bytes()` => `LazyNode::from_bytes()`
2. `sexp_from_bytes()` => `node_from_bytes()`

Reply:
I think (2) makes the most sense. Primarily because of consistency with the rust and python API. I'm not opposed to making the API more like (1), but then I think we also need to change the rust and python APIs (which would be quite involved, since it would spill over onto other code bases)

Reply:
are we keeping this name?

Reply:
Sorry my mistake.
I've just renamed `sexp_from_bytes` to `node_from_bytes`